Update meetings.md (#61)
mod0/csgc-website: csgc-jekyll-src/meetings.md

* *When*: Generally, alternating Mondays, 5:30pm-6:30pm. Meetings for this semester are 9/9, 9/23, 10/7, 10/21, 11/4, 11/18, and 12/2
* *Where*: Torgersen 3160A (If the door is locked, please knock and someone will let you in).

> See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info (**please note this is an open list, and may include faculty**).

Reviewer: This line can go away

## Contact

> * *csgc-g@vt.edu*: Concerns for the Graduate Student Council (**please note this is an open list-serve, and may include faculty**)

Reviewer: ```suggestion
* *csgc-g@vt.edu*: Concerns for the Graduate Student Council (**please note this is an open listserv, and may include faculty**)
```

* *When*: Generally, alternating Mondays, 5:30pm-6:30pm. Meetings for this semester are 9/9, 9/23, 10/7, 10/21, 11/4, 11/18, and 12/2
* *Where*: Torgersen 3160A (If the door is locked, please knock and someone will let you in).

> See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info (**please note some faculty are on the mailing list**).

Reviewer: ```suggestion
See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info (**please note this is an open list, and may include faculty**).
```

Reviewer: I inserted a suggestion using Jamie's wording, since the issue is less that _faculty_ are on the list, and more that it's open to _anyone_.

To answer your question: yes, those are the correct emails, and the new list has been tested.
